app/cafe.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+from app.errors import *
```

Comment:
Ensure that the import order follows the checklist: standard library imports first, followed by local application/library-specific imports. `import datetime` should come before `from app.errors import *`.